daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-eventhubs.md

```diff
@@ -1,12 +1,16 @@
 ---
 type: docs
-title: "Azure Events Hub"
-linkTitle: "Azure Events Hub"
-description: "Detailed documentation on the Azure Event Hubs pubsub component"
+title: "Azure Events Hub (deprecated)"
+linkTitle: "Azure Events Hub (deprecated)"
+description: "Detailed documentation on the Azure Event Hubs pubsub component (deprecated)"
 aliases:
   - "/operations/components/setup-pubsub/supported-pubsub/setup-azure-eventhubs/"
 ---
 
+{{% alert title="Deprecated" color="warning" %}}
+The Azure Event Hubs pubsub component is deprecated and has been removed from starting with release 1.4.
```

Comment:
"The Azure Event Hub pubsub component is deprecated and will no longer be available starting with release 1.4"

Reply:
```suggestion
The Azure Event Hub pubsub component is deprecated and will no longer be available starting with release 1.4.
```